components/cftapps_cluster_lb_backend/main.tf

```diff
@@ -1,6 +1,10 @@
+locals {
+  env = "${(var.env == "aat")? "stg" : "${(var.env == "perftest")? "test" : "${var.env}"}" }"
```

Comment:
Had to add the local variable because of the difference in naming convention between the old and new repo.